setup.py

```diff
@@ -23,7 +23,7 @@ def package_data(pkg, root_list):
 
 setup(
     name='xblock-drag-and-drop-v2',
-    version='2.2.4',
+    version='2.2.6',
```

Comment:
why skip one ?

Reply:
I have released 2.2.5 in last commit but missed to update this file.